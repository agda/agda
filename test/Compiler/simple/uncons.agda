open import Agda.Builtin.Char
open import Agda.Builtin.String
open import Agda.Builtin.Maybe
open import Agda.Builtin.Sigma
open import Common.IO

printTail : String â†’ IO _
printTail str with primStringUncons str
... | just (_ , tl) = putStr tl
... | nothing       = putStr ""

main : _
main = printTail "/test/Compiler/simple/uncons.agda"
