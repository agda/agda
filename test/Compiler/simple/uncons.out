EXECUTED_PROGRAM

ret > ExitSuccess
out > test/Compiler/simple/uncons.agda
